src/modules/axds/commentList/commentList.html

```diff
@@ -1,20 +1,18 @@
 <template>
-    <section>
-        <ul>
-            <template if:true={isLoading}>
-                <axds-comment loading={isLoading} comment={comment}></axds-comment>
-                <axds-comment loading={isLoading} comment={comment}></axds-comment>
-                <axds-comment loading={isLoading} comment={comment}></axds-comment>
-                <axds-comment loading={isLoading} comment={comment}></axds-comment>
-                <axds-comment loading={isLoading} comment={comment}></axds-comment>
-            </template>
-            <template if:false={isLoading}>
-                <template for:each={state.comments} for:item="comment">
-                    <li key={comment.id}>
-                        <axds-comment loading={isLoading} comment={comment}></axds-comment>
-                    </li>
-                </template>
-            </template>
-        </ul>
-    </section>
-</template>
\ No newline at end of file
+  <section>
+    <ul>
+      <template if:true={loading}>
+        <template for:each={skeleton} for:item="i">
```

Comment:
Added the idea of being able to pass a number of skeletons to render here—it defaults to `"5"` in the `commentList.js` file. You set this in `app.js`, where `<axds-comment-list>` is called.